pkg/importer/transport.go

```diff
@@ -102,6 +110,18 @@ func closeImage(src types.ImageSource) {
 	}
 }
 
+func HasPrefix(path string, pathPrefix string) bool {
+	return hasPrefix(path, pathPrefix)
+}
+
+func IsWhiteout(path string) bool {
+	return isWhiteout(path)
+}
+
+func IsDir(hdr *tar.Header) bool {
+	return isDir(hdr)
+}
+
```

Comment:
Let's move these public aliases to new file transport-public.go:

BuildSourceContext
ReadImageSource
HasPrefix
IsWhiteout
IsDir

Also, functions can be aliased as vars:

```
var BuildSourceContext = buildSourceContext
```

Reply:
You are right. I did it